### PR TITLE
feat: [PROD-9525] bold modified parameters

### PR DESCRIPTION
**Pull request by esasova**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

**Review by csm-thu — COMMENTED** [review]
Do you know if it would be possible to move the style definition to the UI components package, and use a common prop (e;g. `dirty`) for all components ? The current solution seems to lead to UI-implementation details being known in the webapp (e.g. the style that contains `'& input'`): it means that changing the UI component implementation would break the style defined in the webapp, whereas it could be a non-breaking change if we use a `dirty` prop instead.

**Comment by esasova** [comment]
Yes, it's possible to apply a class conditionally, good idea, thanks ! Is it better to introduce a new non required prop `isDirty` or we can pass it with all `textFieldProps` ?

**Comment by csm-thu** [comment]
> Yes, it's possible to apply a class conditionally, good idea, thanks ! Is it better to introduce a new non required prop `isDirty` or we can pass it with all `textFieldProps` ?
@esasova a new non-required prop `isDirty` in all generic component would be nice :slightly_smiling_face:

File: src/components/ScenarioParameters/components/ScenarioParametersTabs/ScenarioParameterInput.js [hunk 1]
```diff
@@ -35,9 +35,10 @@ const ScenarioParameterInput = ({ parameterData, context }) => {
   return (
     <Controller
       name={parameterData.id}
-      render={({ field }) => {
+      render={({ field, fieldState }) => {
         const { value: parameterValue, onChange: setRhfValue } = field;
-
+        const { isDirty } = fieldState;
+        const context = { ...tabContext, isDirty };
```
Comment (ardvi) [inline src/components/ScenarioParameters/components/ScenarioParametersTabs/ScenarioParameterInput.js:41]: Same remarq than @csm-thu  , i think is better to add isDirty as a optional prop to all generic input, use it in UI for let basic components apply their particular style.

**Review by ardvi — COMMENTED** [review]
Same remarq as  @csm-thu

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Pull request merged**